requirements.txt

```diff
@@ -9,4 +9,5 @@ geoalchemy2
 osmnx 
 psycopg2
 imageio
-simplification
\ No newline at end of file
+simplification
```

Comment:
the dependency is deleted

Reply:
I am pretty sure that I also already deleted it. I was wondering if it could be that it sneaked back in because of a pull request that was not up to date with the origin.

Reply:
Yes, I also remember you deleted it. 
I was careful in sync every time I create a new branch, but maybe at some point I overlooked